test/python/test_util.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring,no-member
```

Comment:
Can you remove this line? For new files, usually the recommendation is to start without pylint ignores and then add them carefully if there is a strong need for it (and it seems in this case they might not be needed, great!).